.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,44 @@
+name: Bug Report
+description: Create a bug report to help fix issues with OpenTabletDriver
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Fill out the fields below adequately so that we have an easier time solving your issue. Thank you.
+
+  - type: checkboxes
+    id: wiki
+    attributes:
+      label: FAQ
+      description: |
+        Please make sure your issue isn't already covered on the [OpenTabletDriver Wiki](http://opentabletdriver.net/Wiki) and respective FAQ pages.
+      options:
+        - label: I have checked the wiki for my issue
+          required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Sufficiently describe your bug here.
+      placeholder: ex. OpenTabletDriver has stopped detecting my tablet...
+    validations:
+      required: true
```

Probably do the standard `Reproduction steps`, `Expected behaviour`, `Actual behaviour` instead of just one singular "Description" field.

```suggestion
  - type: textarea
    id: steps-to-reproduce
    attributes:
      label: Steps to reproduce
      description: Provide an example of the issue. Pictures, videos, or gifs that showcase the issue are welcome.
      placeholder: |
        Example:
          1. First step
          2. Second step
          3. Issue here
    validations:
      required: true

  - type: textarea
    id: expected-behavior
    attributes:
      label: Expected behavior
      description: Explain what you should expect to happen.
      placeholder: |
        Example:
          "This should happen..."
    validations:
      required: true

  - type: textarea
    id: actual-behavior
    attributes:
      label: Actual behavior
      description: Explain what actually happens.
      placeholder: |
        Example:
          "This happened instead..."
    validations:
      required: true
```